CMakeLists.txt

```diff
@@ -452,6 +452,10 @@ if(cmake_generator_tolower MATCHES "makefile")
     message(STATUS "  PATH        | ${IV_DIR}")
     message(STATUS "  INC         | ${IV_INCLUDE_DIR}")
     message(STATUS "  X11 (INC)   | ${X11_INCLUDE_DIR}")
+    if(IV_ENABLE_X11_DYNAMIC)
+      message(STATUS "              | IV_ENABLE_X11_DYNAMIC ${IV_ENABLE_X11_DYNAMIC}")
+      message(STATUS "              | IV_ENABLE_SHARED ${IV_ENABLE_SHARED}")
+    endif()
```

Comment:
Just nitpick : As we have `Interviews` block, we can remove `if(IV_ENABLE_X11_DYNAMIC)` block and just have:

```
message(STATUS "  Dynamic    | ${IV_ENABLE_X11_DYNAMIC}")
message(STATUS "  Shared     | ${IV_ENABLE_SHARED}")
```

Reply:
I did not want to clutter the status with values that are only very rarely relevant. I anticipate that this ivx11dynam feature will only be used for mac distributions where xquartz is not installed by the user. ie it is only for distribution builds.  Thu unusual aspect of this is that the feature is chosen at the time of the InterViews build and the NEURON build adjusts itself to be consistent
with the feature.

Reply:
Ok, then additional If condition is fine.